src/components/progress-circle/progress-circle.ts

```diff
@@ -1,13 +1,19 @@
 import {
   Component,
-  ChangeDetectionStrategy,
   HostBinding,
+  ChangeDetectorRef,
+  ChangeDetectionStrategy,
   Input
 } from 'angular2/core';
-
+import {isPresent} from 'angular2/src/facade/lang';
 
 // TODO(josephperrott): Benchpress tests.
 
+let DEGREE_IN_RADIANS = Math.PI / 180;
```

Comment:
Also, should all of these maybe be constants?  I only picked this one because it's all caps.